src/odemis/acq/move.py

```diff
@@ -982,7 +1103,8 @@ def run_sub_move(future, component, sub_move):
 
             logging.debug("Performing sub move %s -> %s", component.name, sub_move)
             future._running_subf = component.moveAbs(sub_move)
-        future._running_subf.result(timeout=MAX_SUBMOVE_DURATION)
+        # future._running_subf.result(timeout=MAX_SUBMOVE_DURATION)
+        future._running_subf.result()
```

Comment:
Why is that? Was it for debugging some issue? It probably should be reverted. Or then increase the MAX_SUBMOVE_DURATION.